pact/provider/assertj/src/main/java/org/arquillian/algeron/pact/provider/assertj/PactProviderAssertions.java

```diff
@@ -0,0 +1,29 @@
+package org.arquillian.algeron.pact.provider.assertj;
+
+import org.arquillian.algeron.pact.provider.core.httptarget.Target;
+import org.assertj.core.api.Assertions;
+
+import java.net.URL;
+
+
+public class PactProviderAssertions extends Assertions {
+
+    Target target;
+
+    PactProviderAssertions(Target target) {
+        this.target = target;
+    }
+
+    public static PactProviderAssertions assertThat(Target target) {
+        return new PactProviderAssertions(target);
+    }
+
+    public void verifiesContract() {
+        this.target.testInteraction();
+    }
+
+    public ProviderUrlAssert withUrl(URL url) {
+        return new ProviderUrlAssert(this, url);
```

Comment:
I'm not proposing a removal, but extra method which takes string as parameter, makes URL out of it underneath